app/javascript/gobierto_data/webapp/components/Dataset.vue

```diff
@@ -727,6 +733,13 @@ export default {
       }
     },
     async storeCurrentVisualization(config, opts) {
+      const {
+        params: {
+          queryId
+        }
+      } = this.$route;
+
+      let vizIdFromRoute = +queryId
```

Comment:
Why do you redeclare `queryId`? There aren't more uses

Reply:
To prevent confusion, rename `queryId` to `vizIdFromRoute `, because in this function, we'll try to save or update a Visualization not a Query.